tests/dsl/dtd/dtd_bench_simple_gemm.c

```diff
@@ -4,18 +4,32 @@
 #include "parsec/data_dist/matrix/two_dim_rectangle_cyclic.h"
```

Comment:
Do we really need to have a GEMM test in the runtime itself ? It creates additional dependencies with libraries we don't need for anything else. It might be a good idea to keep the core dependencies at a minimum.

Reply:
I'll make clear that this test is a feature test only and cannot be used to extract performance.

Reply:
Can you change the name of the test so that it doesn't contain the word `bench` in it, like `dtd_cuda_correctness_gemm`

Reply:
Will change for test_dtd_cuda_gemm, to follow the same scheme as the other names

Reply:
Name of the test is still not changed